src/stactools/naip/stac.py

```diff
@@ -207,7 +209,23 @@ def create_item(
             ),
         )
 
-    asset_eo = EOExtension.ext(item.assets["image"])
+    image_asset = item.assets["image"]
+
+    # EO Extension
+    asset_eo = EOExtension.ext(image_asset)
     asset_eo.bands = constants.NAIP_BANDS
 
+    # Raster Extension
+    RasterExtension.ext(image_asset, add_if_missing=True).bands = list(
+        itertools.repeat(
+            RasterBand.create(
+                nodata=0,
+                spatial_resolution=gsd,
+                data_type=DataType.UINT8,
+                unit="none",
```

Comment:
I wasn't able to find any information about scale of offset conversion for these DNs.